File: 백준/Bronze/10988. 팰린드롬인지 확인하기/팰린드롬인지 확인하기.py
word = input()

if len(word) == 1:
    print(1)

i = 0

while i != len(word) // 2:
    if word[i] != word[-(i+1)]:
        print(0)
        break
    elif i == len(word) // 2 - 1:
        print(1)
    
    i += 1

File: 백준/Bronze/10988. 팰린드롬인지 확인하기/팰린드롬인지 확인하기.cc
#include <iostream>
#include <string>
using namespace std;

int main() {
	
	string word;
	cin >> word;

	bool ispalindorme = true;

	for (int i = 0; i < word.length() / 2; i++)
	{
		if (word[i] != word[word.length() - 1 - i])
		{
			ispalindorme = false;
			break;
		}
	}

	if (ispalindorme)
	{
		cout << 1 << endl;
	}
	else {
		cout << 0 << endl;
	}
	
}